docs/index.rst

```diff
@@ -8,7 +8,8 @@ Kagi provides support for WebAuthn_ / FIDO2 security keys and TOTP tokens in Dja
    :caption: Contents:
 
    Home <self>
-   contributing
```

Comment:
I created the `docs/contributing.rst` file as a stub, since there have been times in the past when it's useful to have a somewhat more-concise CONTRIBUTING.rst file (which is used by GitHub) and then a more detailed version in `docs/contributing.rst` that includes the CONTRIBUTING.rst file within it, putting the additional details before and/or after the included CONTRIBUTING.rst file. But that's just an explanation as to why I set it up this way and not necessarily an argument for keeping `docs/contributing.rst`. :smile:

I do wonder, however, whether we can find a way to keep "Contributing" as a top-level element in the primary navigation. I think it would be nice for users who are coming to the docs for the purpose of making a contribution to be able to find the Contributing section as quickly as possible.

Reply:
Yes I did this change only for the TOC sanity. I moved the `include` at the end of the community.rst file. Maybe we can rework the titles to keep contributing as a top level element if you wish.